website_snippet_country_dropdown/README.rst

```diff
@@ -26,41 +50,10 @@ The template have three input text:
 The name of the complete field is customizable when user insert the snippet
 into a form element with the website editor.
 
-Development
```

Comment:
Don't remove all of this. You have to put it inside `USAGE.rst` file with a subtitle level like:
```rst
Development
~~~~~~~~~~~
```